source/simulator/solver.cc

```diff
@@ -841,22 +867,23 @@ namespace aspect
 
         solver_control_cheap.enable_history_data();
         solver_control_expensive.enable_history_data();
+        aspect::internal::InverseWeightedMassMatrix<TrilinosWrappers::PreconditionBase> inverse_weighted_mass_matrix(system_matrix.block(1,1),*Mp_preconditioner,parameters.linear_solver_S_block_tolerance);
```

Comment:
This line could use some line breaks after the type and after each constructor argument